assets/magnify.js

```diff
@@ -3,8 +3,13 @@ function createOverlay(image) {
   overlay = document.createElement('div');
   overlay.setAttribute('class', 'image-magnify-full-size');
   overlay.setAttribute('aria-hidden', 'true');
-  overlay.style.backgroundImage = `url('${image.src}')`;
-  image.parentElement.insertBefore(overlay, image);
+  
+  const bgImg = new Image();
```

Comment:
Nitpick, to try and avoid shortening words so it's easier to read. I think we mostly try to do that in our naming
```suggestion
  const bigImage = new Image();
```